phpcs.xml.dist

```diff
@@ -9,9 +9,13 @@
 		<exclude-pattern>includes/server-timing/object-cache.copy.php</exclude-pattern>
 	</rule>
 
+	<!-- Check up to 20 files simultaneously. -->
+	<arg name="parallel" value="20"/>
```

Comment:
It was added here: https://github.com/WordPress/performance/commit/c00f6bc7d3e97b5588b5a413cb06d4d8735fd21e as part of #953.

We use it in the AMP plugin to speed up PHPCS: https://github.com/ampproject/amp-wp/blob/d4200c4b26446541282aef3c3cc2acd3b93674d7/.phpcs.xml.dist#L187

Reply:
Actually, I tested this and it doesn't seem to help here in Performance Lab. It may be due to the AMP plugin having many more PHP files being tested at once where it then makes an impact. I'm reverting in https://github.com/WordPress/performance/pull/1082.